### PR TITLE
fix: 🐞 UI for synchronized instances and drawer header text layout

### PR DESCRIPTION
**Pull request by themooneer**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description
- UI fix
- Support plural keys for `walletSyncActivated.synchronizedInstances.title`

#### Before 
image
image

#### After
image|
image|

### ❓ Context

- **JIRA or GitHub link**: 
https://ledgerhq.atlassian.net/browse/LIVE-14540

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/Manage/index.tsx [hunk 1]
```diff
@@ -155,12 +155,17 @@ const WalletSyncManage = () => {
...
-                <Text fontWeight="semiBold" variant="large" color="neutral.c100">
+                <Text fontWeight="semiBold" variant="large" color="neutral.c100" flex={1}>
                   {isError
                     ? t("walletSync.walletSyncActivated.synchronizedInstances.error")
-                    : t("walletSync.walletSyncActivated.synchronizedInstances.title", {
-                        count: data?.length,
-                      })}
+                    : t(
+                        data?.length === 1
+                          ? "walletSync.walletSyncActivated.synchronizedInstances.title"
```
[the remark this answers is not in the document]
Comment (themooneer) [reply apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/Manage/index.tsx:163]: it actually works , `title_plural` already exists

[the remark this answers is not in the document]
Comment (cgrellard-ledger) [reply apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/Manage/index.tsx:163]: If `.title` and `.title_plural` exists, you should be able to just do `t("walletSync.walletSyncActivated.synchronizedInstances.title", { count: data?.lendth })`

[the remark this answers is not in the document]
Comment (cgrellard-ledger) [reply apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/Manage/index.tsx:163]: Same as here
Screenshot 2024-10-18 at 14 41 32
Screenshot 2024-10-18 at 14 42 00

[the remark this answers is not in the document]
Comment (themooneer) [reply apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/Manage/index.tsx:163]: hummm ! you right guys, i thoought i was fixing something that actually works ...

**Review by LucasWerey — APPROVED** [review]

**Review by cgrellard-ledger — APPROVED** [review]

**Pull request merged**